config/crd/bases/awx.ansible.com_awxs.yaml

```diff
@@ -73,6 +73,9 @@ spec:
                 type: string
                 maxLength: 255
                 pattern: '^[a-zA-Z0-9][-a-zA-Z0-9]{0,253}[a-zA-Z0-9]$'
+              pg_dump_suffix:
```

Comment:
@mihirlele Thanks for the PR!

An entry is needed for the new `pg_dump_suffix` parameter in the ClusterServiceVersion file.  Here is an example:
* https://github.com/ansible/awx-operator/blob/devel/config/manifests/bases/awx-operator.clusterserviceversion.yaml#L488-L492